# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/2d/PathSkia.h
# Commit: d8c58792b823
# Full Hash: d8c58792b823afb9fe11a1655e6f3e0fcd133510
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2023-04-16 21:18:48
# Regressor Bug: 1821512
# File Overlap Count: 1
# Description:
#   Bug 1821512 - Fix Skia usage for API changes. r=jrmuizel
#   
#   This just tries to address fairly random changes in the Skia API and correct
#   our usage of it in Moz2D and some other places.
#   
# ==============================================================================

diff -r 3427275d765c -r d8c58792b823 gfx/2d/PathSkia.h
--- a/gfx/2d/PathSkia.h	Sun Apr 16 15:34:50 2023 +0000
+++ b/gfx/2d/PathSkia.h	Sun Apr 16 15:34:50 2023 +0000
@@ -91,6 +91,10 @@
 
   Maybe<Rect> AsRect() const override;
 
+  bool GetFillPath(const StrokeOptions& aStrokeOptions,
+                   const Matrix& aTransform, SkPath& aFillPath,
+                   const Maybe<Rect>& aClipRect = Nothing()) const;
+
  private:
   friend class DrawTargetSkia;
 